index.html

```diff
@@ -177,6 +177,47 @@ <h1>About</br>
         </ul>
         <button>Get my resume</button>
       </div>
+
+      <div class="skill-list">        
```

Comment:
**[Required]**

Instead of using _div_, you could have used semantic tags such as _section_ and add _div_'s inside it, like how you did for the About section. 👍